### PR TITLE
William/more lint rules

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204275835641700

### CODE REVIEW

File: src/plugins/borrow-plugins/plugins/aave/AaveNetwork.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export const makeAaveNetworkFactory = (blueprint: AaveNetworkBlueprint): AaveNet
     //
 
     async getAssetPrice(address): Promise<BigNumber> {
+      // eslint-disable-next-line @typescript-eslint/return-await
```
Comment (paullinator) [inline src/plugins/borrow-plugins/plugins/aave/AaveNetwork.ts:92]: why the disable?

File: src/modules/FioAddress/util.ts [hunk 2]
```diff
@@ -788,6 +788,7 @@ export const addBundledTxs = async (fioWallet: EdgeCurrencyWallet | null, fioAdd
   if (fioWallet) {
     try {
       const params = { fioAddress, bundleSets: DEFAULT_BUNDLE_SET_VALUE, maxFee: fee }
+      // eslint-disable-next-line @typescript-eslint/return-await
```
Comment (paullinator) [inline src/modules/FioAddress/util.ts:791]: why the disable?

**Review by paullinator — APPROVED** [review]
Just one small question

File: src/plugins/borrow-plugins/plugins/aave/AaveNetwork.ts (hunk 1 above)
Comment (swansontec) [reply src/plugins/borrow-plugins/plugins/aave/AaveNetwork.ts:92]: The `otherMethods` are `any`, so ESLint doesn't see that the return value is a promise. We know the value is a promise, so we disable the rule.

**Pull request merged**